### PR TITLE
feat: add kpi generators

### LINKED ISSUE #9: Add KPI generators
Per the KPI document:
https://docs.google.com/document/d/1I45VfUnyMmHYm2bVrh7CGpaUK0826fNiIWbF30qS_I0/edit#

Rollups appropriate for the re:dash dashboard should be calculated.

### PR DESCRIPTION
**Pull request by bbangert**
Add's initial batch of KPI generators per Issue #9, along with
CLI script and processor.

Closes #9

@rafrombrc r?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
+# Push Derived Logstreams & Analytics
+
+This repo holds [hindsight](https://github.com/trink/hindsight) log parsing
```
Comment (rafrombrc) [inline README.md]: "log shipping", not "log parsing". The parsing happens elsewhere.

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+## hindsight Processing
+
 Generates Redshift derived streams from Mozilla Push server logs.
+
+## Python Analytics
+
+A Python package `analytics` runs queries for Push analytics and updates rollup
+tables in Redshift.
+
+### Installing
```
Comment (rafrombrc) [inline README.md]: This only covers the python part of things, might want to include info on what's required for the hindsight stuff.

Comment (rafrombrc) [inline README.md]: Alternatively, you could have the top level readme have less info, just refer to the push_load.sh script, and move most of this content down to a readme in the `analytics` folder.

Comment (bbangert) [inline README.md]: I filed a separate ticket for the hindsight stuff, which I assumed you'd be filling in as how it should be run.

**Pull request merged**